src/app/layouts/PrivateLayout.tsx

```diff
@@ -6,11 +6,17 @@ import { axiosInstance } from '@shared/api/axios';
 import { PrivateHeader } from '@widgets/PrivateHeader';
 import { Sidebar } from '@widgets/Sidebar';
 import { BackToButton } from '@widgets/Sidebar/ui/BackToButton';
+import { useCurrenciesQuery } from '@entities/currencies/model/useCurrenciesQuery';
```

Comment:
Add `Public API` and check it everywhere in new entities
```suggestion
import { useCurrenciesQuery } from '@entities/currencies';
```